src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.blobrouter.services.email;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+
+@SpringBootTest
+@ActiveProfiles("db-test")
+public class EmailSenderIntegrationTest {
+
+    private static final String FROM_ADDRESS = "a@b.c";
+    private static final String SUBJECT = "subject";
+
+    @Autowired
+    private EmailSender emailSender;
+
+    @Test
+    public void should_attempt_to_send_email() {
```

Comment:
This test doesn't do anything more than a unit test. I think it would be best to remove it.